apps/studio/components/interfaces/App/FeaturePreview/FeaturePreviewModal.tsx

```diff
@@ -47,13 +45,6 @@ const FeaturePreviewModal = () => {
       properties: { feature: selectedFeatureKey },
       groups: { project: ref ?? 'Unknown', organization: org?.slug ?? 'Unknown' },
     })
-
-    if (selectedFeatureKey === LOCAL_STORAGE_KEYS.UI_TABLE_EDITOR_TABS) {
-      removeTabsByEditor(ref as string | undefined, 'table')
-    }
-    if (selectedFeatureKey === LOCAL_STORAGE_KEYS.UI_SQL_EDITOR_TABS) {
-      removeTabsByEditor(ref as string | undefined, 'sql')
-    }
```

Comment:
I've removed this as we're removing the feature flag anyway and it would be a pain to implement

Reply:
hmm JFYI we aren't removing the feature preview at least not within the next few weeks though as we're doing an incremental default opt in (which means users can still disable the tabs interface if they choose to)

as long as we consider this use case when users are disabling the tabs feature preview

Reply:
> remove tabs when they disable feature flag

yup! that works